release-notes.md

```diff
@@ -15,11 +15,11 @@ can view active developments on GitHub at any time.
 The team is actively working on the multi-node version
 of TimescaleDB which is currently in beta. You can read more about our
 architecture and design for distributed hypertables
-[here](https://docs.timescale.com/beta-v2.0.0/introduction/architecture#distributed-hypertables).
+[here](https://docs.timescale.com/v2.0/introduction/architecture#distributed-hypertables).
```

Comment:
Oops, I just submitted a PR for this section, these will conflict.  

https://github.com/timescale/docs.timescale.com-content/pull/544

One of us can handle the merge later.  I think your later TIP and link to Changes should remain.

Reply:
np. I'll merge again afterwards and deal with it.